src/modules/theme/constants/helpers.js

```diff
@@ -10,3 +10,17 @@ export default {
     text-decoration: none;
   `
 }
+
+export const textTruncator = (text: string, numOfWordsAllowed: number): string => {
+  const ellipsis = '...'
+  const splitTextToWords = text.split(' ')
+  if (splitTextToWords.length < numOfWordsAllowed) {
+    return text
+  }
+
+  const shortenedArrayOfText = splitTextToWords.splice(0, numOfWordsAllowed)
+  const joinShorterendArray = shortenedArrayOfText.join(' ')
+  const textTruncated = joinShorterendArray + ellipsis
+
+  return textTruncated
```

Comment:
I don't really like this solution, but I am not sure if there is a better one...

Reply:
Would be happy to learn a new solution. Please let me know if you have something better

Reply:
I guess this is fine, can't come up with something better.